qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -57,54 +98,38 @@ resources:
 - name: s3.scf-config-sles
   type: s3
   source:
-    endpoint: ((s3-config-endpoint))
-    access_key_id: ((s3-config-access-key))
-    secret_access_key: ((s3-config-secret-key))
-    bucket: ((s3-config-bucket-sles))
+    << : *s3-common-params
     regexp: ((s3-config-prefix-sles))scf-sle-(.*)\.zip$
```

Comment:
Yes, this is because this value is only provided to the `fly set-pipeline` command by the `set-pipeline` script; it doesn't exist in any of our configuration files and is instead determined at pipeline deploy time by the script after the options to `set-pipeline` have been processed

Reply:
may be add a comment there so that we don't use <%=  instead of (( , to avoid rerunning into issues